packages/cosmos_ui_components/lib/components/cosmos_switch.dart

```diff
@@ -1,101 +1,59 @@
 import 'package:cosmos_ui_components/cosmos_theme.dart';
-import 'package:cosmos_ui_components/utils/global_constants.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
-class CosmosSwitch extends StatefulWidget {
+class CosmosSwitch extends StatelessWidget {
   const CosmosSwitch({
     required this.checked,
+    required this.toggleAsset,
+    required this.checkedGradientStops,
+    required this.gradientColors,
```

Comment:
by "parametrizing" i meant to make the values optional. right now its cumbersome to use `CosmosSwitch` as it requires a lot of parameters and it does not work "out of the box". 
```suggestion
   this.checkedGradientStops = [ ... ], 
   this.gradientColors = GlobalConstants.cosmosGradientColors,
```

So a rule of a thumb:
Always provide defaults for the non-essential params for the widgets so that the widget's consumer can easily create widgets with minimal setup, but allow them to alter the values however they want, if needed